.github/workflows/python_build.yaml

```
@@ -77,11 +81,21 @@ jobs:
         run: uv lock
 
       - name: Commit refreshed lockfiles
-        uses: stefanzweifel/git-auto-commit-action@v7
-        with:
-          branch: ${{ github.event.pull_request.head.ref }}
-          commit_message: "fix(ci): refresh package uv lockfiles before python lock checks run"
-          file_pattern: "**/uv.lock"
+        shell: bash
+        run: |
```

actionlint (1.7.11)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)